latch_cli/services/cp.py

```diff
@@ -196,38 +195,37 @@ def _upload_file_chunk(
     return {"ETag": etag, "PartNumber": part_index + 1}
 
 
-def _download(remote_source: str, local_dest: str, executor: cf.ThreadPoolExecutor):
-    """Allows movement of files from Latch -> local machines.
+def download(
+    remote_source: str,
+    local_dest: str,
+    _executor: cf.ThreadPoolExecutor,
+):
+    """Allows movement of files/directories from Latch -> local machines. Called
+    by `cp`.
 
     Args:
         remote_source: A valid path to an existing LatchData file. The path must
-                       be absolute and prefixed with `latch://`.
-        local_dest:    A (relative or absolute) path. If a directory in the path
-                       doesn't exist, that directory and everything following it
-                       becomes the file name - see below.
-
-    This function will initiate a download using an authenticated and presigned
-    URL directly from AWS S3.
-
-    Example: ::
-
-        cp("latch:///sample.fa", "sample.fa")
-
-            Creates a new file in the user's local working directory called
-            sample.fa, which has the same contents as the remote file.
-
-        cp("latch:///dir1/dir2/sample.fa", "/dir3/dir4/sample.fa")
-
-            Creates a new file in the local directory /dir3/dir4/ called
-            sample.fa, which has the same contents as the remote file.
-
-        cp("latch:///sample.fa", "/dir1/doesnt_exist/dir2/sample.fa")
-        # doesnt_exist doesn't exist
-
-            Creates a new file in the local directory /dir1/ called
-            doesnt_exist/dir2/sample.fa, which has the same content as the
-            remote file. Note the nonexistent directory is folded into the
-            name of the copied file.
+            be absolute and prefixed with `latch://`.
+        local_dest: A (relative or absolute) path. If a directory in the path
+            doesn't exist, that directory and everything following it becomes
+            the file name - see below.
+
+    This function will initiate a download using a presigned URL directly from
+    AWS S3.
+
+    Examples:
+        >>> download("latch:///sample.fa", "sample.fa")
+        Creates a new file in the user's local working directory called
+        `sample.fa`, which has the same contents as the remote file.
+
+        >>> download("latch:///dir1/dir2/sample.fa", "/dir3/dir4/sample.fa")
+        Creates a new file in the local directory `/dir3/dir4/` called
+        `sample.fa`, which has the same contents as the remote file.
+
+        >>> download("latch:///sample.fa", "dne/dir2/sample.fa") # dne doesn't exist
+        Creates a new file in the working directory called dne/dir2/sample.fa,
+        which has the same content as the remote file. Note the nonexistent
+        directory is folded into the name of the copied file.
```

Comment:
kind of confusing - do you mean "note that if a specified directory does not exist, it is created"